pywren/wrenhandler.py

```diff
@@ -225,6 +229,13 @@ def generic_handler(event, context_dict, custom_handler_env=None):
         status_key = event['status_key']
         func_key = event['func_key']
         data_key = event['data_key']
+        cancel_key = event['cancel_key']
+
+        # Check for cancel
+        if key_exists(s3_client, s3_bucket, cancel_key):
+            raise Exception("Function cancelled")
```

Comment:
Similar to below can we add a logger.info saying invocation was canceled before start or something like that ?

Reply:
That sounds good, will add it in